api/src/db/migrations/run.py

```diff
@@ -30,13 +33,49 @@ def up(revision: str = "head") -> None:
 
 
 def down(revision: str = "-1") -> None:
+    enable_query_logging()
     command.downgrade(alembic_cfg, revision)
 
 
 def downall(revision: str = "base") -> None:
+    enable_query_logging()
     command.downgrade(alembic_cfg, revision)
 
 
+def enable_query_logging() -> None:
```

Comment:
Testing this, it ends up being very verbose. Is there a way we could make this configurable? Fine if its a follow-up, but locally I want to see just the alembic output, not this so much and now the logs are 3x as long